server/cli-config.yml

```diff
@@ -0,0 +1,57 @@
+# The container name used for the run container
+container-name-run : "eduapp-express-run"
```

Comment:
is this where we include kubernetes, or have we not used them yet?

Reply:
yeah, its the client config; responsible for configs on the cloud.
That line is docker containers related